apps/app/components/home/HeroSection.tsx

```
@@ -167,7 +166,10 @@ export const HeroSection = ({
           }}
         >
           <iframe
-            src={`https://lvpr.tv/?v=${TRANSFORMED_PLAYBACK_ID}&lowLatency=true&backoffMax=1000&ingestPlayback=true&controls=true`}
+            src={getIframeUrl({
+              playbackId: TRANSFORMED_PLAYBACK_ID,
+              lowLatency: "force",
```

This is fine - would just like to point out a couple things:
- If the stream fails to playback in webrtc, it's not going to fallback to HLS
- Which is good right? We do not want HLS playback, since it would be delayed by roughly 30s. But I think there is a case here where we would say "Nah, we prefer having HLS when webrtc is failing, then having a blank player, 30s delay but working video is better than nothing".

Would involve @hthillman and @thomshutt in this decision!

---

Good points, I am inclined towards reverting this back to `lowLatency: true` given its the homepage and better to show a delayed realtime video than nothing.

---

I think I'd actually prefer to just let it fail and iterate on fixing whatever the causes are - like @gioelecerati said, the HLS fallback will be so degraded as an experience in the context of what we're doing here as to be pretty useless